runtime/runtime.py

```diff
@@ -523,6 +575,7 @@ def send_motor_data(data):
     })
 
     # Update sensor values, and send to UI
+    update_hibike()
```

Comment:
Ok Danelle, let me know if you have issues, and tell me when you leave whether you are going to tweak the code or not. I won't be in 101 today, but hopefully someone is to help you.